### PR TITLE
fix(plugins): use explicit unicode strings for plugins

### PR DESCRIPTION
**Pull request by scefali**
I mass converted a bunch of python strings to be explicit unicode so it doesn't cause Py3 issues.

### CODE REVIEW

File: src/sentry_plugins/vsts/client.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class VstsClient(AuthApiClient):
 
     def request(self, method, path, data=None, params=None):
         headers = {
-            "Accept": "application/json; api-version={}".format(self.api_version),
+            u"Accept": "application/json; api-version={}".format(self.api_version),
```
Comment (joshuarli) [inline src/sentry_plugins/vsts/client.py:21]:
```suggestion
            u"Accept": u"application/json; api-version={}".format(self.api_version),
```

Comment (scefali) [reply src/sentry_plugins/vsts/client.py:21]: @joshuarli hahah oops, I put the unicode on the wrong string

Comment (joshuarli) [reply src/sentry_plugins/vsts/client.py:21]: Are you making these changes manually? I'd search for any `format` strings and make sure both the format string and the arguments are unicode. See also https://github.com/getsentry/sentry/pull/21872#discussion_r520068775

Comment (scefali) [reply src/sentry_plugins/vsts/client.py:21]: @joshuarli I was doing changes manually but searching for `format` within `plugins`.

**Review by joshuarli — COMMENTED** [review]
I will leave it to @getsentry/ecosystem for a final approval. I didn't inspect the entire diff.

**Review by MeredithAnya — APPROVED** [review]

**Pull request merged**